Makefile

```diff
@@ -636,41 +636,63 @@ validate-rstudio-image: bin/kubectl
 		continue; \
 	fi; \
 
-# This is only for the workflow action
+# This recipe used mainly from the Pipfile.locks Renewal Action
+# Default Python version
+PYTHON_VERSION ?= 3.11
+ROOT_DIR := $(shell pwd)
+BASE_DIRS := base/c9s-python-$(PYTHON_VERSION) \
+        base/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/minimal/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/datascience/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/pytorch/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/tensorflow/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/trustyai/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/rocm/tensorflow/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/rocm/pytorch/ubi9-python-$(PYTHON_VERSION) \
+		codeserver/ubi9-python-$(PYTHON_VERSION) \
+		runtimes/minimal/ubi9-python-$(PYTHON_VERSION) \
+        runtimes/datascience/ubi9-python-$(PYTHON_VERSION) \
+        runtimes/pytorch/ubi9-python-$(PYTHON_VERSION) \
+        runtimes/tensorflow/ubi9-python-$(PYTHON_VERSION) \
+        runtimes/rocm-tensorflow/ubi9-python-$(PYTHON_VERSION) \
+        runtimes/rocm-pytorch/ubi9-python-$(PYTHON_VERSION)
+
+# Default value is false, can be overiden
+# The below directories are not supported on tier-1
+INCLUDE_OPT_DIRS ?=false
+OPT-DIRS := jupyter/intel/ml/ubi9-python-$(PYTHON_VERSION) \
+			jupyter/intel/pytorch/ubi9-python-$(PYTHON_VERSION) \
+			jupyter/intel/tensorflow/ubi9-python-$(PYTHON_VERSION) \
+			intel/runtimes/ml/ubi9-python-$(PYTHON_VERSION) \
+			intel/runtimes/pytorch/ubi9-python-$(PYTHON_VERSION) \
+			intel/runtimes/tensorflow/ubi9-python-$(PYTHON_VERSION)
```

Comment:
indentation intentional?

Reply:
hm, okay, I don't follow why, though :thinking: you just feel this better readable or am I missing some technical aspect also? :slightly_smiling_face:

Reply:
:P It is because of readability as you mentioned the same as the `BASE_DIRS`
![image](https://github.com/user-attachments/assets/43f95dd5-a763-4412-93f5-8686c0b9acc4)

Reply:
huhhh why the VScode shows them well, but here are so bad?

Reply:
it's a tab vs spaces issue as above - at least that is what I see in my vscode when I fetch your changes locally